.github/workflows/deploy.yml

```diff
@@ -236,18 +236,18 @@ jobs:
           cache-to: type=gha,mode=max
 
       - name: Terraform Setup
-        uses: hashicorp/setup-terraform@v3
+        uses: opentofu/setup-opentofu@v1
         with:
-          terraform_version: ${{ env.tf_version }}
-          terraform_wrapper: false
+          tofu_version: ${{ env.tf_version }}
+          tofu_wrapper: false
 
       - name: Terraform Init
         working-directory: terraform/app
-        run: terraform init -backend-config=${{ matrix.env.environment_name }}.s3.tfbackend
+        run: tofu init -var-file=${{ matrix.env.environment_name }}.tfvars
```

Comment:
Got ahead of myself. The `-var-file` change should've been in 80425243a52a4f4e3e130a3c4625c2343b6d7c55.